Java Basics - Control Task/src/Exceptions/NotValidArgumentException.java

```diff
@@ -0,0 +1,13 @@
+package Exceptions;
+
+public class NotValidArgumentException extends Exception{
```

Comment:
I would rather name InputValidationException

Reply:
Got it